kvstore/ikvstore.go

```diff
@@ -2,31 +2,41 @@
 // SPDX-License-Identifier: Apache-2.0
 package kvstore
 
-import "github.com/veraison/services/config"
+import "github.com/setrofim/viper"
```

Comment:
Avoid personal github's but I am sure this is a Stop gap measure, please raise a guthub issue to remember to revert it back once you have the required fix!

Reply:
Issue raised: https://github.com/veraison/services/issues/44